addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -26,12 +40,11 @@ def _chatbot_crm_prepare_lead_values(self, discuss_channel, description):
 
     def _process_step(self, discuss_channel):
         self.ensure_one()
-
         posted_message = super()._process_step(discuss_channel)
-
         if self.step_type == 'create_lead':
             self._process_step_create_lead(discuss_channel)
-
+        elif self.step_type == "create_lead_and_forward":
+            return self._process_step_create_lead_and_forward(discuss_channel)
```

Comment:
Why not returning the posted_message in this case?

Reply:
We don't want to return the message if there is no operator.
Otherwise `_process_step_forward_operator` is returning the same message as what would super call.